scenes/game/game.gd

```diff
@@ -10,7 +10,6 @@ const Test_StartWithGauge = false
 
 const CardBaseScene = preload("res://scenes/card/card_base.tscn")
 const CardBase = preload("res://scenes/card/card_base.gd")
-const Enums = preload("res://scenes/game/enums.gd")
 const CardPopout = preload("res://scenes/game/card_popout.gd")
```

Comment:
can we actually get rid of all the ones with .gd if we make sure we add class_name to every file?

Reply:
There doesn't seem to be much of a way around doing so for "tscn" files, but will do for all the "gd"s